mongoid.gemspec

```diff
@@ -29,10 +29,14 @@ Gem::Specification.new do |s|
     warn "[#{s.name}] Warning: No private key present, creating unsigned gem."
   end
 
-  s.required_ruby_version     = ">= 2.3"
+  s.required_ruby_version     = ">= 2.5"
   s.required_rubygems_version = ">= 1.3.6"
 
-  s.add_dependency("activemodel", [">= 5.1", "<6.2"])
+  if RUBY_VERSION.start_with?('2.')
+    s.add_dependency("activemodel", [">=5.2", "<6.3"])
```

Comment:
6.2 isn't out yet, so unless I'm missing something the upper bound should remain as < 6.2.

Does 5.1 no longer work on Ruby 2 with the current Mongoid tree?